warehouse/v1/util.js

```diff
@@ -17,7 +20,8 @@ function safeTableName(provider, name = "") {
     tableName = tableName.toLowerCase();
   }
   if (
-    reservedANSIKeywordsMap[provider.toUpperCase()][tableName.toUpperCase()]
+    reservedANSIKeywordsMap[provider.toUpperCase()][tableName.toUpperCase()] &&
```

Comment:
reverse the order of checks to avoid extra check in case of blendo ?

Reply:
done